### PR TITLE
[RMUI+RMTAKAPI] [Feature+Fix] UI configurable assets + Handle LE better

### PR DESCRIPTION
**Pull request by benjamsf**
# What Does This PR Do:
Linked UI PR (by benjamsf)
https://github.com/pvarki/rasenmaeher-ui/pull/24
As well as the pointer to ui, this PR adds to docker compose the RMUI environment table and there the "VITE_ASSET_SET" flag, as well to the example env. Default is "neutral."

Linked RMTAKAPI pr (by Karppo)
https://github.com/pvarki/python-tak-rmapi/pull/61
Now rmtakapi will fetch LE intermediate used to sign dynamically. So either R10 or R11 will be included to truststore provided to ATAK clients.

# Why:
- [The feature request](https://github.com/pvarki/rasenmaeher-ui/issues/20)
TLDR make sure we can have the "FDF" facade with FDF-copyrighted stuff and "neutral" + potentially customisable stuff for other facades.
- [Bug report](https://github.com/pvarki/python-tak-rmapi/issues/60)
TLDR Let's encrypt rotating its intermediates noted us to handle LE intermediates better. Of the TAK family, iTAK and Wintak will trust publicly trusted CAs anyway (which is according to best practices) or in some cases, just disregard some cert issues altogether. So for ATAK we need to ship intermediate to truststore.

### CODE REVIEW

**Comment by benjamsf** [comment]
Adding pointer for https://github.com/pvarki/python-tak-rmapi/pull/61 testing purposes & probably once both are e2e tested, we can merge these together.

**Comment by benjamsf** [comment]
I'll pointer up new UI feature once it has been approved.
